### PR TITLE
edit readme

### PR DESCRIPTION
**Pull request by diw2**
change readme file

### CODE REVIEW

File: jupyter-covid19/Dockerfile [hunk 1]
```diff
@@ -7,15 +7,15 @@ RUN mkdir /home/jovyan/covid19-notebook
 
 ADD https://raw.githubusercontent.com/uc-cdis/covid19-tools/master/covid19-notebooks/jhu-summary-overview/COVID-19-JHU_data_analysis_04072020.ipynb /home/jovyan/covid19-notebook/
 ADD https://raw.githubusercontent.com/uc-cdis/covid19-tools/master/covid19-notebooks/kaggle-demographics/kaggle_data_analysis_04072020.ipynb /home/jovyan/covid19-notebook/
-ADD https://raw.githubusercontent.com/uc-cdis/covid19-tools/master/covid19-notebooks/readme.txt /home/jovyan/covid19-notebook/
+ADD https://raw.githubusercontent.com/uc-cdis/covid19-tools/master/README.md /home/jovyan/covid19-notebook/
```
Comment (paulineribeyre) [inline jupyter-covid19/Dockerfile:10]:
```suggestion
ADD https://raw.githubusercontent.com/uc-cdis/covid19-tools/master/covid19-notebooks/readme.md /home/jovyan/covid19-notebook/
```

File: jupyter-covid19/Dockerfile [hunk 2]
```diff
@@ -7,15 +7,15 @@ RUN mkdir /home/jovyan/covid19-notebook
 
 ADD https://raw.githubusercontent.com/uc-cdis/covid19-tools/master/covid19-notebooks/jhu-summary-overview/COVID-19-JHU_data_analysis_04072020.ipynb /home/jovyan/covid19-notebook/
```
Comment (paulineribeyre) [inline jupyter-covid19/Dockerfile:8]: can we just copy the whole `covid19-tools/master/covid19-notebooks/` folder instead of one by one?

Comment (diw2) [reply jupyter-covid19/Dockerfile:8]: it won't work

Comment (paulineribeyre) [reply jupyter-covid19/Dockerfile:8]: ah right!

File: jupyter-covid19/Dockerfile [hunk 3]
```diff
@@ -7,15 +7,15 @@ RUN mkdir /home/jovyan/covid19-notebook
...
+ADD https://raw.githubusercontent.com/uc-cdis/covid19-tools/master/covid19-notebooks/readme.md /home/jovyan/covid19-notebook/
 ADD https://raw.githubusercontent.com/uc-cdis/covid19-tools/master/covid19-notebooks/requirements.txt /home/jovyan/covid19-notebook/
 ADD https://raw.githubusercontent.com/uc-cdis/covid19-tools/master/covid19-notebooks/chicago-seir-forecast/covid19_seir.ipynb /home/jovyan/covid19-notebook/
 ADD https://raw.githubusercontent.com/uc-cdis/covid19-tools/master/covid19-notebooks/chicago-seir-forecast/Img/seir_diagram.png /home/jovyan/covid19-notebook/
 RUN chmod -R 777 /home/jovyan/covid19-notebook
 RUN touch /home/jovyan/covid19-notebook/COVID-19-JHU_data_analysis_04072020.ipynb
 RUN touch /home/jovyan/covid19-notebook/kaggle_data_analysis_04072020.ipynb
 RUN touch /home/jovyan/covid19-notebook/requirements.txt
-RUN touch /home/jovyan/covid19-notebook/readme.txt
+RUN touch /home/jovyan/covid19-notebook/README.md
```
Comment (paulineribeyre) [inline jupyter-covid19/Dockerfile:18]:
```suggestion
RUN touch /home/jovyan/covid19-notebook/readme.md
```

**Pull request merged**